Tools/jsdoc/cesium_template/static/styles/jsdoc-default.css

```diff
@@ -24,16 +24,25 @@ body {
 code {
   margin: 0;
   padding: 0;
-  font-family:
-    Consolas, "Liberation Mono", Menlo, "Courier New", Courier, Monaco,
-    monospace;
+  font-family: Consolas, "Liberation Mono", Menlo, "Courier New", Courier,
+    Monaco, monospace;
```

Comment:
These were changed because of prettier. Not sure how they escaped getting updated before but this is the way prettier wants them now